zulip_bots/zulip_bots/run.py

```diff
@@ -80,13 +80,9 @@ def exit_gracefully_if_bot_config_file_does_not_exist(bot_config_file: str) -> N
 
 def main() -> None:
     args = parse_args()
-    if os.path.isfile(args.bot):
-        bot_path = os.path.abspath(args.bot)
-        bot_name = os.path.splitext(basename(bot_path))[0]
-    else:
-        bot_path = os.path.abspath(os.path.join(current_dir, 'bots', args.bot, args.bot+'.py'))
-        bot_name = args.bot
-    sys.path.insert(0, os.path.dirname(bot_path))
```

Comment:
The code removed here seems not semantically identical to the new code in `resolve_bot_path`. Can you split this commit into two commits: One where you just move the code as it is, and another where you make the logical changes?